packages/shared-ux/code_editor/mocks/monaco_mock/index.tsx

```diff
@@ -106,7 +106,9 @@ export const MockedMonacoEditor = ({
   className?: string;
   ['data-test-subj']?: string;
 }) => {
-  editorWillMount?.(monaco);
+  useComponentWillMount(() => {
```

Comment:
fixing willMount in the mock because now it re-renders twice due to `setState(editor)` (this is fine, just the tests expectation got broken because of incomplete mock)